setup.py

```diff
@@ -17,6 +17,25 @@
 VERSION = '%d.%d.%d' % (MAJOR, MINOR, MICRO)
 
 
+def read_module(module, package='traitsui'):
+    """ Read a simple .py file from traitsui in a safe way.
+
+    It would be simpler to import the file, but that can be problematic in an
+    unknown system, so we exec the file instead and extract the variables.
+
+    This will fail if things get too complex in the file being read, but is
+    sufficient to get version and requirements information.
+    """
+    base_dir = os.path.dirname(__file__)
+    module_name = package + '.' + module
+    path = os.path.join(base_dir, package, module+'.py')
+    with open(path, 'r') as fp:
```

Comment:
It may be worth specifying the encoding explicitly here, else you end up picking up the "default encoding", which could potentially be something weird that doesn't match the target file.

Reply:
I've modified to read in as utf-8 (even though we write ascii) as that seems safest.